recipe/build.sh

```diff
@@ -1,30 +1,37 @@
 #!/usr/bin/env bash
 
-# @PYTHON@ is used in the build scripts and that breaks witht he long prefix.
-# we need to redefine that to `python`. Note that using
-_PY=$PYTHON
-export PYTHON="python"
+if [[ ${HOST} =~ .*darwin.* ]]; then
```

Comment:
`HOST` is only set by the anaconda compilers which makes the configure line  `--with-libiconv=${LIBICONV}` become `--with-libiconv=`

Reply:
I thought that the `toolchain` was setting that too, another reason to migrate to the new compilers soon :grimacing: 

Thanks!